### ISSUE TITLE
Facebook Error "Circular Redirect Path"

### ISSUE THREAD

**Issue by tordans** [chuckreynolds/WPFBOGP#46]
The Facebook Debug Tool (https://developers.facebook.com/tools/debug/og/object/) gives me the following error for our wordpress pages.

I see this error for the first time so I guess facebook changed their behavior!

We have some logik somewhere in wordpress (probably core, since I cannot find any plugins or custom changes about this) that will always redirect you to the version of an url that ends in "/" -> https://www.betterplace.org/c/neues is redirected to https://www.betterplace.org/c/neues/

The bug:
The the og-URL says ```<meta property="og:url" content="https://www.betterplace.org/c/ueber-uns"/>```(without the trailing slash)

How can I fix this?

![bildschirmfoto 2015-01-09 um 17 47 38](https://cloud.githubusercontent.com/assets/111561/5683387/bd15cdfa-9827-11e4-8f8a-b7a710580d8a.png)

**Comment by chuckreynolds** [comment]
well they're taking the canonical as a redirect in that regard but hokay. so... we messed with the URL a lot in the `develop` branch which MAY provide a better solution to the issue? You could try that and see how it does the URL. 
Here's the commit that's different from `master` and `hotfix`: https://github.com/chuckreynolds/WPFBOGP/commit/269d7c03c2dd7227d3b8db2446500158880b82a0

**Comment by chuckreynolds** [comment]
but it does add one here: https://www.betterplace.org/c/neues/digitale-nothilfe-mal-anders-u25/

Reason I bumped this in the recent commit is a few things. I looked at how other plugins and how wordpress are enforcing trailing slashes on urls... and they do it on non-singular pages... so everything that's not a single post, or a single page. 

Single posts and single pages typically have a link tag rel="canonical" in the header that wordpress puts in. I don't see that on yours but on single pages the trailing slash works as it should.

I'm wondering if you have another plugin or anything in the theme that's causing an issue with the category pages not having a trailing slash? 

....i just noticed this change broke category urls. ugh. I have to look into it more later. I need to sleep.
